app/src/test/java/ch/hikemate/app/authentication/FirebaseAuthRepositoryTest.kt

```diff
@@ -108,6 +110,82 @@ class FirebaseAuthRepositoryTest {
     clearAllMocks()
   }
 
+  @Test
+  fun testCreateAccountWithEmailAndPassword_successful() =
+      runTest(timeout = 5.seconds) {
+        every { mockTask.isSuccessful } returns true
+        every { mockFirebaseAuth.currentUser } returns mockFirebaseUser
```

Comment:
Suggestion: Perhaps move my comment from the `testSignInWithGoogle_successful()` function about mocking the currentUser as successfully logged in before the login even started to this test, since when scrolling through the code this is the first time this mocking happens, so it provides greater understandibility to the code.